FileIO/Legacy/MeshIO.cpp

```diff
@@ -119,6 +123,17 @@ MeshLib::Mesh* MeshIO::loadMeshFromFile(const std::string& file_name)
 		MeshLib::Mesh* mesh (new MeshLib::Mesh(BaseLib::extractBaseNameWithoutExtension(
 		                                               file_name), nodes, elements));
 
+		boost::optional<MeshLib::PropertyVector<int> &> opt_material_ids(
+			mesh->getProperties().createNewPropertyVector<int>(
+				"MaterialIDs", MeshLib::MeshItemType::Cell, 1)
+		);
+		if (!opt_material_ids) {
+			WARN("No material ids read.");
+		} else {
+			MeshLib::PropertyVector<int> & material_ids(opt_material_ids.get());
+			material_ids.resize(materials.size());
+			std::copy(materials.cbegin(), materials.cend(), material_ids.begin());
```

Comment:
stupid question: since ```material_ids``` is empty at this point, wouldn't it be faster to append the mat-vector, e.g.
```material_ids.insert(material_ids.end(), materials.begin(), materials.end());``` ?